2025-05-07 20:56:23,269 - INFO - NVTabular version: 23.08.00
2025-05-07 20:56:23,269 - INFO - cuDF version: 23.04.01
2025-05-07 20:56:23,269 - INFO - Pandas version: 1.5.3
2025-05-07 20:56:23,273 - INFO - Creating preload: dask_cuda.initialize
2025-05-07 20:56:23,273 - INFO - Import preload module: dask_cuda.initialize
2025-05-07 20:56:23,295 - INFO - Run preload setup: dask_cuda.initialize
2025-05-07 20:56:23,504 - INFO -       Start worker at:      tcp://127.0.0.1:41459
2025-05-07 20:56:23,505 - INFO -          Listening to:      tcp://127.0.0.1:41459
2025-05-07 20:56:23,505 - INFO -           Worker name:                          0
2025-05-07 20:56:23,505 - INFO -          dashboard at:            127.0.0.1:39485
2025-05-07 20:56:23,505 - INFO - Waiting to connect to:      tcp://127.0.0.1:40337
2025-05-07 20:56:23,505 - INFO - -------------------------------------------------
2025-05-07 20:56:23,505 - INFO -               Threads:                          1
2025-05-07 20:56:23,505 - INFO -                Memory:                 251.74 GiB
2025-05-07 20:56:23,505 - INFO -       Local Directory: /tmp/dask-worker-space/worker-k87xmotk
2025-05-07 20:56:23,505 - INFO - Starting Worker plugin PreImport-e8da8f26-6215-4734-97df-326cd4572482
2025-05-07 20:56:23,505 - INFO - Starting Worker plugin CPUAffinity-a047c639-6b66-4731-92f1-3e06276c98ec
2025-05-07 20:56:23,506 - INFO - Starting Worker plugin RMMSetup-c28a4165-51fa-48a5-a6e8-9b855ccbee65
2025-05-07 20:56:24,051 - INFO - -------------------------------------------------
2025-05-07 20:56:24,112 - INFO -         Registered to:      tcp://127.0.0.1:40337
2025-05-07 20:56:24,112 - INFO - -------------------------------------------------
2025-05-07 20:56:24,114 - INFO - Starting established connection to tcp://127.0.0.1:40337
2025-05-07 20:56:27,561 - INFO - Run out-of-band function 'clean_worker_cache'
2025-05-07 20:56:29,884 - INFO - Run out-of-band function 'clean_worker_cache'
2025-05-07 22:08:12,043 - INFO - Stopping worker at tcp://127.0.0.1:41459. Reason: worker-close
2025-05-07 22:08:12,048 - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://127.0.0.1:43680 remote=tcp://127.0.0.1:40337>
Traceback (most recent call last):
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once
    event_list = self._selector.select(timeout)
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/home/yuzhuyu/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/home/yuzhuyu/miniconda3/envs/nvtabular_23_08/lib/python3.10/site-packages/distributed/comm/tcp.py", line 269, in write
    raise CommClosedError()
distributed.comm.core.CommClosedError
